packages/types/src/utils/models/ProviderSecrets.ts

```diff
@@ -55,9 +71,24 @@ export interface VSphereProviderSecret {
    */
   password: string;
 
+  /**
+   * Indicate that the client can ignore certifacte verification.
+   *
+   * Provider type: OVirt
+   * Conditions: Optional
+   *
+   * @type {boolean}
+   * @memberof ProviderSecret
+   */
+  insecureSkipVerify?: boolean;
+
   /**
    * VSphere server thumbprint
    *
+   * NOTE: thumbprint is not optional because
```

Comment:
@ahadas hi, can we drop the need for `thumbprint` in the `vsphere` provider if user choose `insecure` ?

Reply:
As per today's meeting, the answer is no.  The vddk process requires a thumbprint even when virt-v2v can be set not verify the server's ssl certificate.

`insecure` just means don't verify the server's SSL certificate

Reply:
:+1: leaving the field required